mate_workspace/src/joystick_module/joystick_module/joystick_demo.py

```diff
@@ -0,0 +1,42 @@
+import pygame
```

Comment:
We can remove this file or call it joystick test or something.

Reply:
Renamed file to joystick_test.py